src/client/public/css/pages/sign-up.css

```diff
@@ -3,3 +3,61 @@
 	font-size: var(--font-size-200);
 	margin: 5px 0 3px 5px;
 }
+
+.wrapper {
+	display: flex;
+	justify-content: center;
+	align-items: center;
+	height: 100vh;
+}
+
+.container {
+	min-width: 675px;
+	box-shadow: 0px 4px 15px 0px rgba(0, 0, 0, 0.25);
+	border-radius: 33px;
+}
+
+.form-top {
+	display: flex;
+	flex-direction: column;
+	align-items: center;
+	border-bottom: 1px solid var(--color-grey-100);
+	padding: 25px 0 15px 0;
+}
+
+.form-top p {
```

Comment:
do not use such selectors, create a classname for that paragraph and use its class as a selector